exampleCourse/questions/element/matching/question.html

```diff
@@ -150,3 +150,45 @@
         </pl-matching>
     </div>
 </div>
+
+<div class="card my-2">
+    <div class="card-body">
+        <pl-question-panel>
+            <p>
+                Example 8: This example uses the default "right" placement for the options.
+            </p>
+        </pl-question-panel>
+            
+        <pl-matching answers-name="ex8" options-placement="right">
+            <pl-statement match="Ottawa">Canada</pl-statement>
+            <pl-statement match="Washington, D.C.">United States</pl-statement>
+            <pl-statement match="Mexico City">Mexico</pl-statement>
+            <pl-statement match="Paris">France</pl-statement>
+            
+            <pl-option>Tijuana</pl-option>
+            <pl-option>Toronto</pl-option>
+            <pl-option>New York City</pl-option>
+        </pl-matching>
+    </div>
+</div>
```

Comment:
I think this example is unnecessary, as it will have a similar output as the examples above. Keep only the example that deviates from the default.